src/iris/ui/static/js/iris.js

```diff
@@ -2803,7 +2808,59 @@ iris = {
       });
     }
   }, // End iris.appStats
-  singlestats:{},
```

Comment:
I would verify that this doesn't break the singlestat endpoint. I think you'll have to add the singlestat base_route as an exception like you did unsubscribe.